packages/components/src/components/sidebar/sidebar-item.tsx

```diff
@@ -22,20 +46,199 @@ export class SidebarItem {
     }
   }
 
+  handleEventEmission() { 
+    this.ifxSidebarItem.emit({value: this.value, component: this.el})
```

Comment:
this.value is never set and always ""

Reply:
It is emitted when you click on an item. 
Check function **toggleSubmenu()**, which is invoked on click. 
 `onClick={() => this.toggleSubmenu()}`